### PR TITLE
[SQLite3] Use generic TLP Where oracle

### PR DESCRIPTION
**Pull request by malwaregarry**
rebased from #941

### CODE REVIEW

**Review by mrigger — APPROVED** [review]
LGTM. I guess we can merge this?

File: src/sqlancer/common/oracle/TLPWhereOracle.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        this.state = state;
+        this.gen = gen;
+        this.errors = expectedErrors;
+    }
+
+    @Override
+    public void check() throws SQLException {
+        S s = state.getSchema();
+        AbstractTables<T, C> targetTables = TestOracleUtils.getRandomTableNonEmptyTables(s);
+        gen = gen.setTablesAndColumns(targetTables);
```
Comment (mrigger) [inline src/sqlancer/common/oracle/TLPWhereOracle.java:43]: Not related to this PR, but I was a bit confused by this. The `set` somewhat suggests that we set a value in the existing generator, but actually, we create a new generator. Can we reflect this in the naming of the methods? E.g., `newWithTablesandColumns` or `copyWithTablesAndColumns` or so?

Comment (malwaregarry) [reply src/sqlancer/common/oracle/TLPWhereOracle.java:43]: yup i can do that after the other oracles have been created

**Comment by malwaregarry** [comment]
@mrigger sorry for the late response. We can merge this. 

This is rebased from the this PR https://github.com/sqlancer/sqlancer/pull/941

**Pull request merged**